components/modal-clubs.jsx

```diff
@@ -0,0 +1,295 @@
+var React = require('react/addons');
```

Comment:
all the code here was extracted from [`pages/clubs.jsx`](https://github.com/mozilla/teach.webmaker.org/blob/develop/pages/clubs.jsx) as well